docs/development/doc_guide.rst

```diff
@@ -1,76 +1,633 @@
-************************
-Documentation Guidelines
-************************
+*******************
+Documentation Guide
+*******************
 
-This document describes the documentation requirements and guidelines
-to be followed during the development of PlasmaPy and affiliated
-packages.
+Documentation that is up-to-date and understandable is vital to the
+health of a software project. This page describes the documentation
+requirements and guidelines to be followed during the development of
+PlasmaPy and affiliated packages.
+
+`PlasmaPy's online documentation`_ is hosted by `Read the Docs`_ and is
+available at these locations.
+
+* The documentation corresponding to the most recent official release
+  is labelled ``stable`` and is found at
+  `https://docs.plasmapy.org/ <https://docs.plasmapy.org/>`_ or
+  `https://docs.plasmapy.org/en/stable/ <https://docs.plasmapy.org/en/stable/>`_.
+
+* The documentation corresponding to the ``main`` branch on
+  `PlasmaPy's GitHub repository`_ is labelled ``latest`` and can be
+  found at `https://docs.plasmapy.org/en/latest/
+  <https://docs.plasmapy.org/en/latest/>`_.
+
+A preview of the documentation is generated every time a pull request
+is created or updated. You can access this preview by scrolling down
+to the checks at the bottom of a pull request, and clicking on
+``Details`` next to ``docs/readthedocs.org:plasmapy``.
```

Comment:
I like this bit, but it could be made clearer by adding a screenshot from an actual PR.

Reply:
That's a good idea for a first contribution.  I'll create an issue for it.